README.md

```diff
@@ -388,16 +387,6 @@ HOME=~/spacemacs emacs
 Note: If you're using the Fish shell, then you'll need to modify the last
 command to: `env HOME=$HOME/spacemacs emacs`
 
-### Modify spacemacs-start-directory variable
-This solution is better suited to "embed" Spacemacs into your own configuration.
-If you've cloned Spacemacs into `~/.emacs.d/spacemacs/`, then drop the following
-lines in the `~/.emacs.d/init.el` file:
-
-```elisp
-(setq spacemacs-start-directory "~/.emacs.d/spacemacs/")
-(load-file (concat spacemacs-start-directory "init.el"))
-```
-
```

Comment:
Setting `spacemacs-start-directory` in `init.el` should be ok, however I think that we still need to document how this would be setup. In this case we would need to mention that it is required to load the spacemacs init.el for this to work.